### ISSUE TITLE
Preview of laser is off

### ISSUE THREAD

**Issue by materschief** [Direwolf20-MC/MiningGadgets#263]
When Mining with the laser the preview is off. depending on the angle it disappears completeley. I am playing TNP Limitless 8. Screeshot shows my bug.
![javaw_2024-12-25_18-23-22](https://github.com/user-attachments/assets/79fe51ef-3cf3-487a-954a-be2c1eff5277)

Below is an Screenshot with me pointing further to the right. 
![javaw_2024-12-25_18-26-33](https://github.com/user-attachments/assets/d05c4957-cf72-4705-a642-8412034321f6)

Version of Mining Gadget is 1.18.6

**Comment by MichaelHillcox** [comment]
Running shaders?

**Comment by materschief** [comment]
Nope, it has has some installed, but not active. The pack has Sodium though. Can that be the cause?

**Comment by MichaelHillcox** [comment]
Due to the lack of issue templates on this repo. We're missing essential information for us to be able to resolve your issue. This in combination with the time gap between the creation date and this response means that reproduce this issue is little to none.

If you are still having this issue, please open a new issue on our github using our "Bug Report" template
